app/services/recording-mode.ts

```diff
@@ -180,12 +191,22 @@ export class RecordingModeService extends PersistentStatefulService<IRecordingMo
     }, 10 * 1000);
   }
 
-  addRecordingEntry(filename: string) {
+  /**
+   * Add entry to recording history and show notification
+   * @param filename - name of file to show in recording history
+   * @param showNotification - primarily used when recording in dual output mode to only show the notification once
+   */
+  addRecordingEntry(filename: string, display?: TDisplayType) {
     const timestamp = moment().format();
     this.ADD_RECORDING_ENTRY(timestamp, filename);
+
+    const message = display
+      ? $t(`A new ${capitalize(display)} Recording has been completed. Click for more info`)
```

Comment:
I believe translation team would like `capitalize(display)` to be interpolated so that it can be translated. 
e.g `$t('A new %{displayType} Recording...', { displayType: $t(display /* or capitalize(display) */) }`

Reply:
Done